vnext/Microsoft.ReactNative.Cxx/UI.Xaml.impl.override.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+// In this file we override xaml::Application::Current() to return nullptr
+// instead of crashing when Application is not found.
+#pragma once
+#include <winrt/base.h>
+
+#ifndef USE_WINUI3
+
+#ifndef WINRT_Windows_UI_Xaml_2_H
+#include <winrt/impl/Windows.UI.Xaml.2.h>
+
+namespace winrt::impl {
+
+template <>
+WINRT_IMPL_AUTO(Windows::UI::Xaml::Application)
```

Comment:
Yes, I think in this case having a helper would make more sense. I was just played too much with the idea how I can augment the generated consumer classes with custom code instead of having all these manually written ReactContext, ReactDispatcher, and other classes.

---
In reply to: [432988186](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r432988186) [](ancestors = 432988186)

Reply:
Defined TryGetCurrentApplication instead.

---
In reply to: [433249993](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r433249993) [](ancestors = 433249993,432988186)